bigtable/client/mutations.h

```diff
@@ -115,6 +115,18 @@ class SingleRowMutation {
     }
   }
 
+  /// Create a single-row multiple-cell mutation from a variadic list.
+  template <typename... M>
+  SingleRowMutation(std::string row_key, M&&... m)
```

Comment:
Please make this constructor `explicit`.

Reply:
constructor is now explocit, fixed.